projects/xpdf/project.yaml

```diff
@@ -1,4 +1,5 @@
 homepage: "https://www.xpdfreader.com/"
+main_repo: 'https://gitlab.freedesktop.org/freetype/freetype'
```

Comment:
wrong repo

Reply:
Removed because they only have a website to download the latest code.